fuel-toolchain.toml

```diff
@@ -2,5 +2,5 @@
 channel = "latest-2023-10-17"
```

Comment:
The version of forc you are using doesn't update unless this channel does

Reply:
Updated